ykman/descriptor.py

```diff
@@ -137,7 +137,7 @@ def list_drivers(transports=sum(TRANSPORT)):
                 yield dev
 
 
-def open_driver(transports=sum(TRANSPORT), serial=None, pid=None, attempts=3):
```

Comment:
Can we make sleep_time here grow exponentially instead of linearly? With that the number of attempts can probably be lowered.